flow/arguments.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"encoding/json"
+	"github.com/onflow/cadence"
+	jsoncdc "github.com/onflow/cadence/encoding/json"
+)
+
+type CadenceArgument struct {
+	Value cadence.Value
+}
+
+func (v CadenceArgument) MarshalJSON() ([]byte, error) {
```

Comment:
Good idea to create a wrapper value and implement the json marshalling interfaces 👍

Reply:
All the credit goes to @psiemens 🙇‍♂️